ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java

```diff
@@ -271,7 +273,23 @@ void updateAccountsFromFile(String accountJsonPath) throws IOException {
     long startTime = System.currentTimeMillis();
     Collection<Account> accountsToUpdate = getAccountsFromJson(accountJsonPath);
     if (!hasDuplicateAccountIdOrName(accountsToUpdate)) {
-      if (accountService.updateAccounts(accountsToUpdate)) {
+      Collection<Account> allAccounts = accountService.getAllAccounts();
+      // Update the snapshot version to resolve conflict
+      Map<Short, Account> existingAccountsMap = new HashMap<>();
```

Comment:
since we may not always want to avoid the version check, maybe keep the default behavior the same and add a --force option that will allow unconditional overrides

Reply:
added a new option --ignoreSnapshotVersion